src/codewhisperer/util/closingBracketUtil.ts

```diff
@@ -39,6 +42,15 @@ export async function handleExtraBrackets(
     endPosition: vscode.Position,
     startPosition: vscode.Position
 ) {
+    /**
+     * recommendation provided in the function argument is the original recommendation, whereas here what we need is "effective recommendation"
+     * for example given file context ('|' is where we trigger the service)
+     * anArray.pu|
+     * recommendation returned: "sh(element);"
+     * typeahead: sh(
+     * the effective recommendation "written" by CodeWhisperer: element);
+     */
+    recommendation = editor.document.getText(new vscode.Range(startPosition, endPosition))
```

Comment:
this overwrites the `recommendation` parameter. doesn't that mean the `recommendation` parameter should be removed?

Reply:
yea make sense, i originally thought there might be some other use case of that parameter in the future thus keeping it. But we can add it back if needed, let's remove it.